index.html

```diff
@@ -86,6 +86,13 @@
           date: "2025-02-25",
           publisher: "W3C"
         },
+        "credman-accessibility": {
+          title: "5.3. Credential Selection",
+          href:" https://w3c.github.io/webappsec-credential-management/#user-mediated-selection",
+          authors: ["Nina Satragno", "Marcos Cáceres"],
+          date: "2024-08-13",
+          publisher: "W3C"
+        }
```

Comment:
Oopsy... there's a different way to do this... 

```suggestion
```

Reminds me I need to add support for citing sections to respec.